aiohttp/web_request.py

```diff
@@ -409,7 +409,8 @@ def post(self):
                     out.add(field.name, ff)
                 else:
                     value = yield from field.read(decode=True)
-                    if content_type.startswith('text/'):
+                    if content_type is None or \
+                            content_type.startswith('text/'):
```

Comment:
I agree with @kxepal, we should not decode data if we do not know content-type
it would very hard to reason about exception if one occurs from this code